### ISSUE TITLE
Can't open a PDF 

### ISSUE THREAD

**Issue by benoitrosa** [oxplot/pdfrankenstein#15]
Hi,

Trying this software, I get the following error in the current branch : 

```
pdfrankenstein: failed to open  (filename)
qpdf: unknown option --warning-exit-0

Usage: qpdf [options] infile outfile
For detailed help, run qpdf --help
```

Will try with the latest release and update here

**Comment by oxplot** [comment]
Can you please put down what command you ran and if possible, the input PDF file you used.

**Comment by benoitrosa** [comment]
Hi,

It happens with any PDF I have, with the current branch as well as the v0.9.1 release. I was just pushing the button in the GUI to open a PDF. I'm on Ubuntu 20.04.4 for what it's worth.

**Comment by oxplot** [comment]
Oh I missed the error. I believe you need a more recent qpdf version. The version you have doesn't seem to support the `--warning-exit-0` flag.

**Comment by oxplot** [comment]
Looks like you need at least qpdf 10.0.2

EDIT: [updated readme](https://github.com/oxplot/pdfrankenstein/commit/a14769962c171c0c9dfd071eadc6deddec968cb1) to reflect this.

**Comment by benoitrosa** [comment]
Along the same lines, I had to install libgtk-3-dev for compiling.

But actually, with qpdf 9.1.1 from the official repos I can load the PDFs if I remove the --warning-exit-0 flag in the session.go file. I also needed to update inkscape using the inkscape.dev/stable ppa (not sure about the required version)

**Comment by oxplot** [comment]
Thanks. I [opened an issue](https://github.com/oxplot/pdfrankenstein/issues/16) to build packages for other distros.

> if I remove the --warning-exit-0 flag in the session.go file

Keep in mind that will cause qpdf to fail for some PDFs.

**Comment by oxplot** [comment]
> I also needed to update inkscape using the inkscape.dev/stable ppa

What Inkscape version did you originally have and what error did you encounter?

**Comment by benoitrosa** [comment]
Yes I'm sure it's there for a reason, I was just experimenting with the software ;) 

For Inkscape I had the basic one shipped with 20.04, I think it's 0.91. I couldn't open any document due to a CLI option (--pdf-pages=X I think).
